test/eth/TransactionManager.spec.js

```diff
@@ -163,3 +163,103 @@ test('lifecycle hooks', async () => {
   const wipe = cdp.wipeDai(1);
   await Promise.all([txMgr.confirm(wipe), mineBlocks(service)]);
 });
```

Comment:
Added separate tests for give and bite, I could create a new describe block for them along with the pre-existing tests if that makes more sense.

Reply:
Yeah, definitely [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) things up a bit.